tests/utils.py

```diff
@@ -6,8 +6,24 @@
 """
 Shared test utilities.
 """
+import sys
+
+from types import FrameType
 
 from structlog._log_levels import _NAME_TO_LEVEL
 
 
 stdlib_log_methods = [m for m in _NAME_TO_LEVEL if m != "notset"]
+
+
+_REAL_GETFRAME = sys._getframe
+
+
+def mock_getframe(__depth: int = 0) -> FrameType:
```

Comment:
could https://github.com/hynek/structlog/blob/main/tests/additional_frame.py be used somehow?

Reply:
I think it can, I will give it a try.

Reply:
Works well, thanks for the hint, removed `mock_getframe`.